### DISCUSSION TITLE
Any way to reduce code range when using autocode? Is there way to set a code count cap?

### DISCUSSION THREAD

**Discussion by cowwithoutleg** [ccbogel/QualCoder#898] [Q&A]
Is there a way to cap the upper limit of code counts? I'm using auto code to find certain keywords in a set of large texts. Some codes have more 1000 hits, but others just a a few. So, my range is around 1-1000. This washes out the heatmap I want to use, so I was thinking of setting a upper count limit of 50 to make my code range 1-50.  If there are more than 50 code matches, it just hits the 50 limit, so my range would be 1-50+. I hope this makes sense, it's kind of hard to explain, but is there anyway to do this other than manually going through each text and coding it without auto code?

**Comment by ccbogel** [comment]
good idea.  I have added that feature in to the latest code. I have not tested on large codings, as I did not have 50 + codings. but it should work. Please let me know. As if it works , I could further edit the number options. currently 50, 100, 150 etc. maybe smaller intervals might be better.
